pkg/render/fluentd.go

```diff
@@ -1063,10 +1060,7 @@ func (c *fluentdComponent) eksLogForwarderDeployment() *appsv1.Deployment {
 
 	// Determine the namespace in which Linseed is running. For managed and standalone clusters, this is always the elasticsearch
 	// namespace. For multi-tenant management clusters, this may vary.
-	linseedNS := ElasticsearchNamespace
-	if c.cfg.Tenant.MultiTenant() {
-		linseedNS = c.cfg.Tenant.Namespace
-	}
+	linseedNS := LinseedNamespace(c.cfg.Tenant)
```

Comment:
We don't need to declare an intermediate variable here now that we have a helper function